common/reviews/api/rush-lib.api.md

```diff
@@ -501,6 +515,21 @@ export interface IGetChangedProjectsOptions {
 export interface IGlobalCommand extends IRushCommand {
 }
 
+// @internal
+export interface _ILastInstallFlagJson {
```

Comment:
This is a lot of state to be exposing via the API.  As far as I can tell, the entire purpose of these `@internal` API's is to support this code:

https://github.com/microsoft/rushstack/blob/5d9c506caec86b1d2b979703c893b67481451bb5/apps/rush/src/RushVersionSelector.ts#L33-L37

...which reads one file that looks like this:

**C:\Users\YourAlias\.rush\node-v18.19.1\rush-5.120.1\last-install.flag**
```js
{
  "node": "18.19.1"
}
```

Simply carving out a tiny copy of the relevant logic would eliminate this `@internal` API entirely.  It is not a breaking change, because **RushVersionSelector.ts** is importing from a direct dependency of `@microsoft/rush`, whose `rush-lib` version is exactly equal to the `rush` version.

Reply:
Also, do we even need `"node": "18.19.1"` if the path includes `node-v18.19.1`? 😁

Reply:
Is the suggestion to remove the @internal flag?